### ISSUE TITLE
Exploratory Testing

### ISSUE THREAD

**Issue by MehnDaniel** [BME-MIT-IET/iet-hf2021-iet-csipcsap#8]

**Comment by MehnDaniel** [comment]
We are conducting an exploratory testing on the API, during which we examine the calls' functions regarding the product it was created for, and the products it could be implemented into in the future.

**Comment by MehnDaniel** [comment]
The API has a swagger documentation, which had the following problems:
- The Vehicle model's owner property was presumably misspelled as "ownder"
- The documented responses are inconsistent regarding the "Unauthorized" response, as some of the calls don't have it

The Vehicle model also has 3 properties, called "x, y, z". These represent the 3 dimensions of the cargo hold. I would consider renaming these properties to something like "cargoX, cargoY, cargoZ".

**Comment by MehnDaniel** [comment]
While testing the API, I ran into a problem with gathering data about the package. The cause of the error was a wrong argument in a mapToDouble call. 

The error has been fixed in the ExploratoryTestMD branch.

Pull request created with the title:
**getPackagesWightSum fix in Transfer.java**
(The typo in the function name is made when creating the pull request, not present in the function.)
Comments in the pull request contain additional information.
